app/Services/Manager.php

```diff
@@ -37,9 +47,9 @@ public function exportCSV($model, $reportbacks = false)
         $users = $model->users;
 
         $headers = ['northstar_id', 'first_name', 'last_name', 'email', 'cell'];
+
         if ($reportbacks) {
```

Comment:
I can update to `hasReportback` I was concerned about this to when I needed the extra check that we got a reportback back, it's a little repetitive now. That might make it more readable.